src/ImageSharp/PixelFormats/PixelOperations{TPixel}.PixelBenders.cs

```diff
@@ -0,0 +1,68 @@
+﻿// <copyright file="PixelOperations{TPixel}.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.PixelFormats
+{
+    using System.Numerics;
+    using System.Runtime.CompilerServices;
+    using ImageSharp.PixelFormats.PixelBlenders;
+
+#pragma warning disable CS1710 // XML comment has a duplicate typeparam tag
+    /// <summary>
+    /// A stateless class implementing Strategy Pattern for batched pixel-data conversion operations
+    /// for pixel buffers of type <typeparamref name="TPixel"/>.
+    /// </summary>
+    /// <typeparam name="TPixel">The pixel format.</typeparam>
+    public partial class PixelOperations<TPixel>
+#pragma warning restore CS1710 // XML comment has a duplicate typeparam tag
+        where TPixel : struct, IPixel<TPixel>
+    {
```

Comment:
The single pixel operations are actually on the `PixelBlender<TPixel>` classes themselves